server/database/users.go

```diff
@@ -11,3 +11,11 @@ func (m *MongoDB) CreateNewStudent(student middleware.Student) error {
 	_, err := collection.InsertOne(ctx, student)
 	return err
 }
+
+func (m *MongoDB) FindStudent(login middleware.LoginRequest) bool {
+	ctx := context.TODO()
+	collection := m.mongo.Database(USER_DATABASE).Collection(STUDENTS_COLLECTION)
+	result := collection.FindOne(ctx, login)
```

Comment:
can you rename this to "err"

Reply:
it does not return err

Reply:
https://pkg.go.dev/go.mongodb.org/mongo-driver/mongo#Collection.FindOne